src/components/domain/rightSection/index.js

```diff
@@ -0,0 +1,89 @@
+import { getPostById, postPost, putPost } from "../../../api/posts.js";
+import { getItem, setItem } from "../../../storage/localStorage.js";
+import Editor from "./Editor.js";
+import { debounce } from "../../../utils/utils.js";
+
+class RightSection {
+  constructor({ $target, initialState }) {
+    this.$target = $target;
+    this.state = initialState;
+    this.TEMP_POST_SAVE_KEY = `temp-post-${this.state ? this.state.id : "new"}`;
+    this.DELAY_TIME = 500;
+
+    this.$rightSection = document.createElement("div");
+    this.$rightSection.className = "right-section";
+    this.editor = new Editor({
+      $target: this.$rightSection,
+      initialState,
+      onEditing: (post) => {},
+    });
+    this.$target.appendChild(this.$rightSection);
```

Comment:
관련 있는 코드끼리는 서로 붙어있도록 작성하는 게 코드를 파악하기 쉬워지고, 가급적이면 추상화 수준을 맞춰주는 것도 많은 도움이 됩니다.

```suggestion
    this.$rightSection = document.createElement("div");
    this.$rightSection.className = "right-section";
    this.$target.appendChild(this.$rightSection);
    
    this.editor = new Editor({
      $target: this.$rightSection,
      initialState,
      onEditing: (post) => {},
    });
```